internal/cmd/git-combine/git-combine.go

```diff
@@ -112,7 +113,7 @@ func Combine(path string, opt Options) error {
 
 		for depth := 0; depth < opt.LimitRemote; depth++ {
 			if time.Since(lastLog) > time.Second {
-				logger.Printf("Collecting new commits... (remotes %s, commit depth %d)", remote, depth)
+				logger.Info(fmt.Sprintf("Collecting new commits... (remotes %s, commit depth %d)", remote, depth))
```

Comment:
```suggestion
				logger.Info("Collecting new commits", log.String("remotes", remote), log.String("commit_depth", depth))
```

Doing a Printf when building a log message is usually a bad thing, because we're missing out on the ability to search for the attribute value. It probably doesn't matter much in the present case, but it also doesn't cost us anything to use attributes there.